compiler/nvrtc_builder.cc

```diff
@@ -20,10 +20,9 @@ namespace chainer_compiler {
 namespace {
 
 std::string CleanseIdent(const std::string& s, const char* prefix = "v_") {
-    std::locale loc;
     std::string o = prefix;
     for (char c : s) {
-        if (std::isalnum(c, loc)) {
+        if (std::isalnum(c)) {
```

Comment:
Could you use the two-argument version? I decided to use the default `locale`, which is classic locale instead of system's.